web/src/index.css

```diff
@@ -1,14 +1,312 @@
-body {
-  margin: 0;
-  padding: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+/* Bootstrap helper classes */
+.m-0 {
+  margin: 0 !important;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.mt-0,
+.my-0 {
+  margin-top: 0 !important;
 }
+
```

Comment:
overwrite default style of materialize not looks great idea

Reply:
Where did you see it?

Reply:
It's just css helper classes for margin and padding.

Reply:
i think better to use something like a grid
and default indent